### PR TITLE
(cdburnerxp) move to manual

### LINKED ISSUE #2427: (cdburnerxp) suggest moving the package to manual
### Discussed in https://github.com/orgs/chocolatey-community/discussions/2402

<div type='discussions-op-text'>

<sup>Originally posted by **TheCakeIsNaOH** January  9, 2024</sup>
The cdburnerxp update script is currently failing due to a cerificate error. 
As per https://cdburnerxp.se/impressum the company behind it has shut down, so it is unlikely to be updated in the future. 
Thus, I think moving the package to be manual would make sense.</div>

### PR DESCRIPTION
**Pull request by TheCakeIsNaOH**
## Description

Moves the `cdburnerxp` files to the manual directory.

## Motivation and Context

Fixes #2427 

## How Has this Been Tested?

N/A, moving package files only

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- N/A The added/modified package passed install/uninstall in the Chocolatey Test Environment(https://github.com/chocolatey-community/chocolatey-test-environment/). _Note that we don't support the use of any other environment_.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/cdburnerxp/update.ps1
Comment (AdmiringWorm) [inline automatic/cdburnerxp/update.ps1:1]: Don't delete the update file. This is still useful if we want to create a fix version, or if development suddenly is resumed at some point (although that is unlikely).

Any file called `update.ps1` is executed when a manual package is pushed through AppVeyor.

File: manual/cdburnerxp/cdburnerxp.nuspec
Comment (AdmiringWorm) [inline manual/cdburnerxp/cdburnerxp.nuspec:1]: Can you also update the `packageSourceUrl` for the package?

I don't remember whether we update it automatically or not for manual packages (I don't think we do).

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]

File: manual/cdburnerxp/cdburnerxp.nuspec
Comment (TheCakeIsNaOH) [reply manual/cdburnerxp/cdburnerxp.nuspec:1]: Done.

File: automatic/cdburnerxp/update.ps1
Comment (TheCakeIsNaOH) [reply automatic/cdburnerxp/update.ps1:1]: Done.

**Review by AdmiringWorm — APPROVED** [review]
LGTM, failure on AppVeyor is a known problem for manual packages.

**Comment by AdmiringWorm** [comment]
@TheCakeIsNaOH your changes have been merged. Thanks for your contribution 👍

**Pull request merged**